package.json

```diff
@@ -24,49 +24,54 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-auto-import": "^1.2.21",
-    "ember-cli-babel": "^7.7.3",
-    "git-repo-version": "1.0.2",
-    "lodash.merge": "~4.6.1",
-    "rollbar": "~2.5.5"
+    "ember-auto-import": "^1.5.3",
+    "ember-cli-babel": "^7.13.0",
+    "ember-cli-htmlbars": "^4.2.0",
+    "git-repo-version": "^1.0.2",
+    "lodash.merge": "^4.6.1",
+    "rollbar": "^2.14.4"
```

Comment:
Note I bumped the rollbar version and am now using `^`

Reply:
Is rollbar following semver? :) Im afraid if they introduce breaking change and the addon will not handle that correctly.

Reply:
Good point, I didn't consider that 😅

In that case, I'd encourage using a peer dependency for rollbar. That way, consuming ember apps would be able to set the version of rollbar they want to use.

That also pushes the concern of breaking changes in rollbar to consuming apps too

Reply:
It is pushing the concern, however, we will have to force all users to add it and maintain the compatible version which still will be locked to eg `~2.14.0` as we cannot guarantee if this addon will be compatible with `2.15.0` without testing.

Let's leave other packages, but I would suggest keeping rollbar with `~` as a compromise between `^` and strict version. So we will put some trust in it but not all. What do you think?

Reply:
Ok, sounds good 👍